packages/expo-router/src/modal/ModalComponent.tsx

```diff
@@ -0,0 +1,22 @@
+import { NavigationContext } from '@react-navigation/native';
+import { View } from 'react-native';
+
+import { type ModalConfig } from './ModalContext';
+
+interface ModalComponentProps {
+  modalConfig: ModalConfig;
+}
+
+export function ModalComponent({ modalConfig }: ModalComponentProps) {
+  const component = modalConfig.component;
+  const navigationProp = modalConfig.parentNavigationProp;
+
+  if (navigationProp) {
```

Comment:
Add a comment indicating when this would be the case. My understanding is that Expo Router is always wrapped in a navigation context.